data/persons.txt

```diff
@@ -1850,3 +1849,200 @@ ship "Modified Ladybug"
 	explode "huge explosion" 10
 	"final explode" "final explosion large"
 	description `This is an old war-torn Hai Ladybug with numerous modifications done on the hull. Apart from the additional turret mounts, the main adjustments on the vessel are three huge engine cells - two in the back replacing the old engine pods, and one in the front replacing the gun mounts. These push the Modified Ladybug to speeds only an insane captain would enjoy.`
+
+
+person "Patrol Team"
+	government "Author"
+	frequency 300
+	personality
+		mining harvests plunders
+	phrase
+		word
+#		Marksman
+			"If you're interested in helping out but don't have time, consider being a proofreader! You can take my job, and I get to laze around doing nothing!"
+			"The number of times someone breaks the fourth wall, it's mind-boggling!"
+			"Look, we aren't here for your money. We do all this for free and look where that gets us."
+			"If you're interested in helping out but don't have time, consider looking through the proposals on GitHub! Sometimes, a new perspective is needed!"
+			"What do you mean you don't have a save file for this pull request!?"
+			"This statement is, intentionally wrote wrongly to make point."
+
+#		Saugia
+			"Yeah... way too much detail. I can't see the cockpit anymore."
+			"Just don't go ham on details."
+			"These areas are looking much better, though. The contrast between the less detailed area here and the more detailed areas elsewhere is a lot nicer."
+			"I'm just trying to see this half that size and I'm losing a lot of detail."
+			"It's awesome, but far too detailed."
+			"Now there's a lot less detail conflicting with one another."
```

Comment:
```suggestion
			"Sometimes, a large amount of detail on a model can look less appealing."
			"Sometimes, a small amount of detail on a model can look more appealing."
			"Make sure to look at an image from a few steps back as well."
			"Down with tangents, up with overlaps."
			"Don't be afraid of adding more contrast."
			"If you want to learn how to create artwork, start with a donut."
```
Kinda just some generic ones I'd say or believe in ~~as opposed to just grabbing every time I talk about detail.~~.